marklogic-data-hub-central/ui/src/components/results-tabular-view/results-tabular-view.test.tsx

```diff
@@ -47,4 +46,23 @@ describe("Results Table view component", () => {
         // Check for Empty Table
         expect(getByText(/No Data/i)).toBeInTheDocument();
     });
+
+    test('Array data renders properly', () => {
+        const {  getByText, queryByText } = render(
+                <ResultsTabularView
+                    data={entitySearch.results}
+                    entityPropertyDefinitions={entityPropertyDefinitions}
+                    selectedPropertyDefinitions={selectedPropertyDefinitions}
+                    columns={[]}
+                    hasStructured={false}
+                />
+        )
+        // Check table column headers are rendered
+        expect(queryByText('Carmdin')).toBeNull();
+        expect(queryByText('Carm din')).toBeNull();
+        expect(getByText('Carm')).toBeInTheDocument();
+        expect(getByText('din')).toBeInTheDocument();
+        expect(getByText('Carm')).toContainHTML('<div style="text-overflow: ellipsis; overflow: hidden;">Carm</div>');
+        expect(getByText('din')).toContainHTML('<div style="text-overflow: ellipsis; overflow: hidden;">din</div>');
+    });
```

Comment:
both "Carm" and "din" can be in different cells and still the test would pass. We want to verify that both Carm and din are in the same cell. Try using:
expect(getByText('Carm').closest('td')).toEqual(getByText('din').closest('td'))

Reply:
Added. Thank you.